registration/src/main/java/com/kyriba/registration/api/AttendeeController.java

```diff
@@ -0,0 +1,77 @@
+package com.kyriba.registration.api;
+
+import com.kyriba.registration.api.dto.AttendeeRegistrationRequest;
+import com.kyriba.registration.domain.RegistrationStatus;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import lombok.Value;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_UTF8_VALUE;
+
+
+@Api(value = "Attendee registration endpoint")
+@RestController
+@RequestMapping("/api/v1/attendees")
+public class AttendeeController
+{
+  /**
+   * UCS: Attendee: 1. register in a conference
+   *
+   * @param attendee input API DTO for new attendee registration
+   * @return registered attendee output API DTO
+   */
+  @ApiOperation(value = "Register new attendee", response = AttendeeRegistrationResponse.class)
+  @PostMapping(produces = APPLICATION_JSON_UTF8_VALUE, consumes = APPLICATION_JSON_UTF8_VALUE)
+  AttendeeRegistrationResponse register(
+      @ApiParam(value = "Attendee registration object", required = true) @RequestBody AttendeeRegistrationRequest attendee)
```

Comment:
You could move produces and consumes configuration to class level.

Reply:
yes, but method 
`StatusResponse status(@ApiParam(value = "Attendee identity", required = true) @PathVariable String id)` 
does not consumes JSON.